bicep/network-new.bicep

```diff
@@ -159,6 +159,9 @@ var nsg_rules = {
     // INBOUND RULES
     //
 
+    // Allow ssh from Frontend to compute
+    AllowSshFrontendComputeIn: ['200', 'Inbound', 'Allow', 'Tcp', 'Ssh', 'subnet', 'frontend', 'subnet', 'compute']

```

Comment:
When I tried these rules out, I got the following error

InvalidTemplate","message":"Deployment template validation failed: 'The template variable 'securityRules' is not valid: The language expression property 'frontend' doesn't exist, available properties are 'cyclecloud, compute, lustre, bastion'

Reply:
updated to cyclecloud subnet name